cbmc_viewer/markup_summary.py

```diff
@@ -12,14 +12,17 @@
 ################################################################
 # Data passed to jinja to generate proof summary from summary.jinja.html
 
-VALID_SUMMARY_DATA = voluptuous.schema_builder.Schema({
+VALID_SUMMARY_DATA = voluptuous.Schema({
     # coverage section of the summary
     'coverage': {
-        'overall': {
-            'percentage': float, # percentage of lines hit
-            'hit': int, # lines hit
-            'total': int # lines total
-        },
+        'overall': voluptuous.Any(
+            voluptuous.Schema({
```

Comment:
I don't think `Schema`s need to be specified recursively like this. Passing just the two dicts to `Any` would be fine. I think you only need recursive schemas when the constructors to each of them differ, for example, if for one of them `required=True` and the other one `required=False`. 

In fact, it seems like you probably do want `required=True` for every key in this `"overall"` dict, right? So either the inner `Schema` should be omitted, or you should pass `required=True` to the inner constructor also.

Reply:
Done. Thanks